### PR TITLE
React native video audiofocus callback param 

### PR DESCRIPTION
**Pull request by pengcao1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by pengcao1** [comment]
anybody can review this PR?

**Comment by pengcao1** [comment]
@tdreyno 
@kamal 
@leemhenson 
could kindly please help to review this PR?

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]
Marking as changes requested until @rbuckton's comment is addressed.

File: types/react-native-video/index.d.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ export interface VideoProperties extends ViewProps {
   onPlaybackStalled?(): void;
   onPlaybackResume?(): void;
   onPlaybackRateChange?(): void;
-  onAudioFocusChanged?(): void;
+  onAudioFocusChanged?(event: any): void;
```
[the remark this answers is not in the document]
Comment (pengcao1) [reply types/react-native-video/index.d.ts]: I'm afraid not.
that defined by Android system

**Pull request closed without merging**